mkdocs.yml

```diff
@@ -18,17 +18,26 @@ nav:
       - Service Registry:
         - HTTP:
           - device-discovery: api/serviceregistry/device-discovery-generic-http.md
+          - system-discovery: api/serviceregistry/system-discovery-generic-http.md
         - MQTT:
-          - device-discovery: api/serviceregistry/device-discovery-generic-mqtt.md          
+          - device-discovery: api/serviceregistry/device-discovery-generic-mqtt.md
+          - system-discovery: api/serviceregistry/system-discovery-generic-mqtt.md          
     - Data Models:
       - AddressDesriptor: api/data-models/address-descriptor.md
+      - DeviceDesriptor: api/data-models/device-descriptor.md
```

Comment:
DeviceDes**C**riptor. You probably copied my typo from above: AddressDes**C**riptor :)
Please correct both.